docs/en/security/trash.md

```diff
@@ -38,41 +38,41 @@ juicefs status META-URL
 
 :::note
-UTC is used when naming directories in the first level.
+The first level directory is named after the UTC time.
 :::
 
 :::tip
-You can use `juicefs info` to check inode of a file or directory.
+You can use `juicefs info` to check inode of a file or a directory.
 :::
 
-### Access
+### Privileges
 
-All users are allowed to browse the trash and see the full list of removed files. However, files in trash keep their original modes, so users can only read files that they can read before. If JuiceFS is mounted with `--subdir <dir>` (mostly used as a CSI driver on Kubernetes), the whole trash will be hidden and can't be accessed.
+All users are allowed to browse the trash and see the full list of removed files. However, since JuiceFS keeps the original modes of the trashed files, users can only read files that they can read before. If JuiceFS is mounted with `--subdir <dir>` (mostly used as a CSI driver on Kubernetes), the whole trash will be hidden and can't be accessed.
 
-It is not permitted to create files within the trash. Deleting or purging a file are forbidden as well for non-root users, even if he/she is owner of this file.
+It is not permitted to create files inside the trash. Deleting or purging a file are forbidden as well for non-root users, even if he/she is the owner of this file.
 
 ### Recover/Purge
 
-It is suggested to ask root user to recover files, since root is allowed to move them out of trash with a single `mv` command, and causes no data copy. Other users, however, can only recover a file by reading its content and write it to another new file.
+It is suggested to recover files as root, since root is allowed to move them out of trash with a single `mv` command without any extra data copy. Other users, however, can only recover a file by reading its content and write it to another new file.
 
-JuiceFS client will check the trash every hour and purge old entries. At least one active client is required to make it happen. Like recovering, only root user is allowed to manually purge entries by `rm` command.
+Since it is JuiceFS client which is in charge of checking the trash every hour and purging old entries, you need at least 1 active client to perform checking and purging. Like recovering, only root user is allowed to manually purge entries by `rm` command.
```

Comment:
```suggestion
Since it is JuiceFS client which is in charge of checking the trash every hour and purging old entries, you need at least ONE active client. Like recovering, only the root user is allowed to manually purge entries by the `rm` command.
```